scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -300,6 +301,25 @@
          (catch Exception e
            (log/error e "Mesos scheduler status update error")))))
 
+(defn write-sandbox-url-to-datomic
+  "Takes a sandbox file server URL from the compute cluster and saves it to datomic."
+  [conn task-id sandbox-url]
+  (try (let [db (db conn)
```

Comment:
Nitpick: add a newline after `try`?

Reply:
hostname is the node hostname and is not usable for the sandbox url